backend/app/main.py

```diff
@@ -85,4 +88,35 @@ async def upload_url(page_url: str):
       detail={"error": type(e).__name__, "message": str(e)}
     )
 
-  return result
\ No newline at end of file
+  return result
+
+@app.delete("/delete/document")
+async def delete_document(filename_or_url: str):
+  try:
+      ids = await delete_document_by_prefix(filename_or_url)
+      return {
+      "status": "ok",
+      "message": "Document deleted",
+      "Document id:s deleted": ids
+    }
+  except Exception as e:
+    # Return 400 Bad Request
+    raise HTTPException(
+      status_code=status.HTTP_400_BAD_REQUEST,
+      detail={"error": type(e).__name__, "message": str(e)}
+    )
+
+@app.get("/get_uploaded_ids")
```

Comment:
rename to "/vector/ids"
```suggestion
@app.get("/vector/ids")
```